### PR TITLE
Services section

### PR DESCRIPTION
**Pull request by alanbiju79**
| <a href="https://ellipsis.dev" target="_blank">Ellipsis</a> | :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit dc3c9d1f327d1d17eb53d1b026061008fa136c59.  | 
|--------|--------|

### Summary:
This PR updates the `/docs/features/cookie-categories.mdx` file, replacing the 'Cookie Details' section with a 'Services' section and changing the document title to 'Cookie Categories & Services'.

**Key points**:
- Updated `/docs/features/cookie-categories.mdx` file.
- Replaced 'Cookie Details' section with 'Services' section.
- Changed document title to 'Cookie Categories & Services'.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: docs/features/cookie-categories.mdx [hunk 1]
```diff
@@ -10,9 +10,14 @@ Cookie Categories are used to organize your cookies into groups. You can create
...
 - Analytics
 - Marketing
+- Personalization
 
 Users will be consenting to these groups of cookies. Each group will have a toggle switch to allow users to opt-in or opt-out of the category.
 
-## Cookie Details
+## Services
 
-Cookie Details are used to describe the cookies used on your website. The required fields are the cookie name and category. The other fields are optional but recommended. You can use CookieChimp's [AI Auto-fill](/docs/ai) feature to automatically fill in the cookie details.
```
Comment (danielpaul) [inline docs/features/cookie-categories.mdx:18]: Do we still want to keep something around cookie details saying how they can use AI etc?

File: docs/features/cookie-categories.mdx [hunk 2]
```diff
@@ -10,9 +10,14 @@ Cookie Categories are used to organize your cookies into groups. You can create
...
+Services are used to group cookies associated to a company or service.
+Cookies can be directly added to a cookie category or you can group them into services.
+You can create as many services as you need.
+Services must be associated with a category and each service will have a toggle inside it's category section to allow users to opt-in or opt-out of the service.
```
[the remark this answers is not in the document]
Comment (danielpaul) [reply docs/features/cookie-categories.mdx:22]: Hmm don't know it should be its... let's check this out

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: docs/features/cookie-categories.mdx (hunk 1 above)
Comment (alanbiju79) [reply docs/features/cookie-categories.mdx:18]: Seems a bit out of place here, there's already the section in the AI doc

File: docs/features/cookie-categories.mdx [hunk 3]
```diff
@@ -10,9 +10,14 @@ Cookie Categories are used to organize your cookies into groups. You can create
...
 
-## Cookie Details
+## Services
 
-Cookie Details are used to describe the cookies used on your website. The required fields are the cookie name and category. The other fields are optional but recommended. You can use CookieChimp's [AI Auto-fill](/docs/ai) feature to automatically fill in the cookie details.
+Services are used to group cookies associated to a company or service.
+Cookies can be directly added to a cookie category or you can group them into services.
+You can create as many services as you need.
+Services must be associated with a cookie category and each service will have a toggle inside its category section to allow users to opt-in or opt-out of the service.
+You can then use the services attribute to [block scripts](/docs/block-scripts-cookies).
```
Comment (danielpaul) [inline docs/features/cookie-categories.mdx:23]:
```suggestion
Cookies can be directly added to a cookie category or you can group them into services. Services must be associated with a cookie category and each service will have a toggle inside its category section to allow users to opt-in or opt-out of the service. You can then use the services attribute to [block scripts](/docs/block-scripts-cookies).
```

**Review by danielpaul — APPROVED** [review]

**Pull request merged**